cxx14.cmake

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2013-2017, Ruslan Baratov
+# Copyright (c) 2017, Richard Hodges
+# All rights reserved.
+
+if(DEFINED POLLY_CXX14_CMAKE_)
+    return()
+else()
+    set(POLLY_CXX14_CMAKE_ 1)
+endif()
+
+# Don't use polly_init (no generator expected)
```

Comment:
There are few requirements for toolchains in Polly, attachment to CMake generator is one of them.